templates.html

```diff
@@ -0,0 +1,203 @@
+                    <label for="q-statement-input">Question Statement</label>
+                    <textarea class="form-control" id="q-statement-input" rows="3" placeholder="Question Statement"></textarea>
+                </fieldset>
+                <div class="js-question-type-area multiple-choice"></div>
+            </form>
+        </div>
+
+        <div class="js-hints">
+            <fieldset class="form-group hints"> <label>Hints </label>
+                <button type="button" class="btn btn-default hint js-hint hint-1 js-hint-active active" data-hint="1"><span>Hint 1</span></button><button
+                        type="button" class="btn btn-default hint js-hint hint-2" data-hint="2"><span>2</span></button><button
+                        type="button" class="btn btn-default hint js-hint hint-3" data-hint="3"><span>3</span></button>
+                <textarea class="form-control js-hints-textarea" rows="3" placeholder="Hint 1"></textarea>
+            </fieldset>
+        </div>
+    </div>
+
+    <div class="js-modal-question-mc">
+        <div class="js-mc-add-remove">
+            <fieldset class="form-group mc-option-change">
+                <button type="button" class="btn btn-default mc-remove-option js-mc-remove-option">
+                    <span class="glyphicon glyphicon-minus"></span>
+                </button><button
+                    type="button" class="btn btn-default mc-add-option js-mc-add-option">
+                    <span class="glyphicon glyphicon-plus"></span>
+                </button>
+            </fieldset>
+        </div>
+
+        <div class="js-mc-options">
+            <fieldset class="form-group mc-text-option js-mc-text-option">
+                <div class="input-group js-option">
+                    <div class="input-group-addon correct-answer">
+                        <button type="button" class="btn btn-default mc-correct-answer">
+                            <span class="glyphicon glyphicon-ok"></span>
+                        </button>
+                    </div>
+                </div>
+            </fieldset>
+        </div>
+    </div>
+
+    <div class="js-modal-question-code">
+        <div class="js-io">
+            <fieldset class="form-group input-output">
+                <label>Input / Output </label>
+                <button type="button" class="btn btn-default js-remove-io">
+                    <span class="glyphicon glyphicon-minus"></span>
+                </button><button
+                    type="button" class="btn btn-default js-add-io">
+                    <span class="glyphicon glyphicon-plus"></span>
+                </button><button
+                    type="button" class="btn btn-default js-io-btn js-io-active active" data-io="1">
+                    <span>IO 1</span>
+                </button><button
+                    type="button" class="btn btn-default js-io-btn " data-io="2">
+                    <span>2</span>
+                </button><button
```

Comment:
Is a span needed? You can avoid it and obtain the same result (unless the CSS is doing something weird).